docs/03-tutorials/00-api-exposure/apix-02-expose-and-secure-service.md

```diff
@@ -110,22 +110,59 @@ Follow the instructions in the tabs to deploy an instance of the HttpBin service
   HttpBin - secure endpoints of a service
   </summary>
 
-1. Deploy an instance of the HttpBin service:
+1. Deploy an instance of the HttpBin service in your Namespace:
 
-  ```shell
-  kubectl apply -f https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml
+  ```bash
+  kubectl -n ${NAMESPACE_NAME} create -f https://raw.githubusercontent.com/istio/istio/master/samples/httpbin/httpbin.yaml
+  ```
+
+2. Export these values as environment variables:
+
+  ```bash
+  export NAMESPACE={NAMESPACE_NAME} #If you don't have a Namspeace yet, create one.
+  export TLS_SECRET={SECRET_NAME} #e.g. use the TLS_SECRET from your Certificate CR i.e. httpbin-tls-credentials.
+  export WILDCARD={WILDCRAD_SUBDOMAIN} #e.g. *.api.mydomain.com
+  export DOMAIN={CLUSTER_DOMAIN} #This is a Kyma domain or your custom subdomain e.g. api.mydomain.com.
   ```
 
-2. Expose the service and secure it by creating an APIRule CR:
+3. Create a Gateway CR. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: networking.istio.io/v1alpha3
+   kind: Gateway
+   metadata:
+     name: httpbin-gateway
+     namespace: $NAMESPACE
+   spec:
+     selector:
+       istio: ingressgateway # Use Istio Ingress Gateway as default
+     servers:
+       - port:
+           number: 443
+           name: https
+           protocol: HTTPS
+         tls:
+           mode: SIMPLE
+           credentialName: $TLS_SECRET
+         hosts:
+           - "$WILDCARD"
+   EOF
+   ```
+
+4. Expose the service and secure it by creating an APIRule CR in your Namespace:
```

Comment:
These steps are nearly identical to the instructions in "Expose a service", and again the Note box at the bottom is the same. Only here we add the words "and secure it" and of course the code sample is different.

Is there any way we could reduce redundancy in these instructions, in order to make it clearer for readers what's the difference between those step instructions (and to reduce maintenance effort for TWs)?

Reply:
I will think about what we can do about that, thanks!

Reply:
@mmitoraj lets talk about it

Reply:
I talked to Magda and the main focus here is the simplification of the multiple domain tutorial. We will have a follow-up to do some consolidation of the two remaining tutorials a little bit later.